### ISSUE TITLE
Failed to start the session because headers have already been sent

### ISSUE THREAD

**Issue by IdahoPL** [bacula-web/bacula-web#127]
New installation of bacula-web 8.6.1 from FreeBSD ports. PHP 7.4.32.

```
# su -m www -c "cd /usr/local/www/bacula-web && php bwc setupauth"
It's now time to setup the application back-end database

Please note that all informations stored in the user database will be destroyed
Can we proceed ?

Answer (Yes/No): Yes
Let's go !
Deleting application back-end database
Creating database schema
PHP Fatal error:  Uncaught RuntimeException: Failed to start the session because headers have already been sent by "/usr/local/www/bacula-web/console.php" at line 183. in /usr/local/www/bacula-web/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php:145
Stack trace:
#0 /usr/local/www/bacula-web/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php(352): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->start()
#1 /usr/local/www/bacula-web/vendor/symfony/http-foundation/Session/Session.php(261): Symfony\Component\HttpFoundation\Session\Storage\NativeSessionStorage->getBag()
#2 /usr/local/www/bacula-web/vendor/symfony/http-foundation/Session/Session.php(283): Symfony\Component\HttpFoundation\Session\Session->getBag()
#3 /usr/local/www/bacula-web/vendor/symfony/http-foundation/Session/Session.php(69): Symfony\Component\HttpFoundation\Session\Session->getAttributeBag()
#4 /usr/local/www/bacula-web/core/db/Database.php(47): Symfony\Component\HttpFoundation\Session\Session->has()
#5 /usr/local/www/bacula-web/core/db/DatabaseFactory.php(30): Core\Db\Database->__construct()
#6 /usr/local/www/bacula-web/console.php(217): Core\Db\DatabaseFactory::getDatabase()
#7 /usr/local/www/bacula-web/bwc(21): require_once('/usr/local/www/...')
#8 {main}
  thrown in /usr/local/www/bacula-web/vendor/symfony/http-foundation/Session/Storage/NativeSessionStorage.php on line 145
```

**Comment by dfranco** [comment]
Thanks for the bug report @IdahoPL 

I'll try to reproduce it and fix it asap.

**Comment by IdahoPL** [comment]
@dfranco: Thank you. If you need more details or access to VM with this problematic setup let me know.

**Comment by dfranco** [comment]
I can reproduce the issue, I'll produce a fix and release another minor version

**Comment by dfranco** [comment]
Issue #128 created, already working on it

**Comment by dfranco** [comment]
Hi @IdahoPL ,

The issue has been resolved and the fix is available in the new release v8.6.2.

Thanks again for your bug report.

Do not hesitate to open a bug report or feature request here if needed.

Please note also that you can use [discussion](https://github.com/bacula-web/bacula-web/discussions) if you have a question or feedback.

Best regards
